src/Controls/src/Core/Binding.cs

```diff
@@ -120,7 +122,13 @@ internal override void Apply(object context, BindableObject bindObj, BindablePro
 			object src = _source;
 			var isApplied = IsApplied;
 
-			base.Apply(src ?? context, bindObj, targetProperty, fromBindingContextChanged, specificity);
+			var bindingContext = src ?? Context ?? context;
```

Comment:
Previously we didn't include `Context` into the call to `Apply`. Is there a case which we could break with this change?

Reply:
this is correct as is